lib/commands/web.js

```diff
@@ -8,6 +9,44 @@ let extensions = {};
 
 Object.assign(extensions, iosCommands.web);
 
+extensions.getExtraNativeWebTapOffset = async function () {
+  let offset = 0;
+
+  // first try to get tab offset
+  try {
+    let el = await this.findNativeElementOrElements('-ios predicate string', `name LIKE '*, Tab' AND visible = 1`, false);
+    // this height is either there or not, so cache
+    if (!util.hasValue(this._webTabBarHeight)) {
+      el = util.unwrapElement(el);
+      const height = (await this.getNativeRect(el)).height;
+      log.debug(`Found tabs with height ${height}.`);
+      this._webTabBarHeight = height;
+    }
+    offset = this._webTabBarHeight;
+  } catch (ign) {
+    // no element found, so no tabs and no need to deal with offset
+  }
+
+  // next try to see if there has been scrolling
+  try {
+    await this.findNativeElementOrElements('accessibility id', 'ReloadButton', false);
+    // reload button found, which means scrolling has not happened
+  } catch (err) {
+    try {
+      let el = await this.findNativeElementOrElements('accessibility id', 'URL', false);
+      // this height can vary depending on the scrolling, so compute
+      el = util.unwrapElement(el);
+      const height = (await this.getNativeRect(el)).height;
```

Comment:
No. That value changes depending on the scrolling, and the only way to tell which one of the two values it could be it is, is to check the value.